drivers/vdpa/virtio/virtio_vdpa.c

```diff
@@ -583,11 +434,25 @@ virtio_vdpa_virtq_enable(struct virtio_vdpa_priv *priv, int vq_idx)
 		return -EINVAL;
 	}
 
-	ret = virtio_vdpa_virtq_doorbell_relay_enable(priv, vq_idx);
-	if (ret) {
-		DRV_LOG(ERR, "%s virtq doorbell relay failed ret:%d",
-						priv->vdev->device->name, ret);
-		return ret;
+	if (priv->configured){
+		virtio_vdpa_lcore_id = rte_get_next_lcore(virtio_vdpa_lcore_id, 1, 1);
```

Comment:
We get this lcore_id using rte_get_next_lcore every time. So do we need to define it as a global var? maybe temp var?

Reply:
we use global to prevent use same core next time

Reply:
ok